ci/ray_ci/bisect/linux_validator.py

```diff
@@ -0,0 +1,83 @@
+import time
+
+from pybuildkite.buildkite import Buildkite
+
+from ci.ray_ci.bisect.validator import Validator
+from ci.ray_ci.utils import logger
+from ray_release.test import (
+    Test,
+    LINUX_TEST_PREFIX,
+)
+from ray_release.aws import get_secret_token
+from ray_release.configs.global_config import get_global_config
+
+BUILDKITE_ORGANIZATION = "ray-project"
+BUILDKITE_POSTMERGE_PIPELINE = "postmerge"
+BUILDKITE_BUILD_RUNNING_STATE = [
+    "creating",
+    "scheduled",
+    "running",
+]
+BUILDKITE_BUILD_PASSING_STATE = [
+    "passed",
+    "skipped",
+]
+BUILDKITE_BUILD_FAILING_STATE = [
+    "failing",
+    "failed",
+    "blocked",
+    "canceled",
+    "canceling",
+    "not_run",
+]
+TIMEOUT = 2 * 60 * 60  # 2 hours
+WAIT = 10  # 10 seconds
+
+
+class LinuxValidator(Validator):
+    def _get_buildkite(self) -> Buildkite:
+        buildkite = Buildkite()
+        buildkite.set_access_token(
+            get_secret_token(get_global_config()["ci_pipeline_buildkite_secret"]),
+        )
+
+        return buildkite
+
+    def _get_rayci_select(self, test: str) -> str:
+        test = Test.gen_from_name(f"{LINUX_TEST_PREFIX}{test}")
```

Comment:
[1]

Reply:
is this going to be the one thing that is different from windows? or are there other logic that is going to be different too?

Reply:
probably, i want to defer windows later, i need to run a bisect job for a window test first and see what problems come up, but i can also make this generic first for linux